pkg/controllers/constants/constants.go

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
can we move this to pkg/constants/constants.go?

Reply:
Yeah that makes sense, done